src/legacy/core_plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -292,6 +292,11 @@ export const esaggs = (): ExpressionFunction<typeof name, Context, Arguments, Re
         const cleanedColumn: KibanaDatatableColumn = {
           id: column.id,
           name: column.name,
+          _meta: {
```

Comment:
any ideas for naming this ?

Reply:
meta sgtm, but I think I would remove the `_`